docs/source/reference/pysupported.rst

```diff
@@ -934,7 +934,8 @@ The following built-in functions are supported:
   cannot be a function type (e.g. ``getattr(numpy, 'cos')`` is not supported as
   it returns a function type).
 * :func:`hash` (see :ref:`pysupported-hashing` below)
-* :class:`int`: only the one-argument form
+* :class:`int`: support `int`, `float`, or `str` as the first argument,
+  and support an optional `base` argument with `str`
```

Comment:
Depending on how `int` overflow ends up being resolved in this PR, the docs might need associated limitations noting.